src/DwitTech.AccountService.Core/Services/ValidationService.cs

```diff
@@ -0,0 +1,100 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Utilities;
+using DwitTech.AccountService.Data.Entities;
+using DwitTech.AccountService.Data.Context;
+using System.Net.Mail;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ValidationService : IValidationService
```

Comment:
Do we really need this class? ActivationService should handle everything to do with UserActivation

Reply:
> Do we really need this class? ActivationService should handle everything to do with UserActivation

Resolved